### ISSUE TITLE
error: the following arguments are required: -src_vocab/--src_vocab

### ISSUE THREAD

**Issue by ishaansharma** [OpenNMT/OpenNMT-py#1938]
I am using this script from [docs/source/examples/Library.ipynb](https://github.com/OpenNMT/OpenNMT-py/blob/7a93327bcc6465a631c91fbd0356420a0fc33c05/docs/source/examples/Library.ipynb) as I am trying to build a custom script just like [examples/library/minimal_transformer_training.py](https://github.com/OpenNMT/OpenNMT-tf/blob/master/examples/library/minimal_transformer_training.py)

After executing 

```
base_args = (["-config", "toy-ende/config.yaml", "-n_sample", "10000"])
opts, unknown = parser.parse_known_args(base_args)
```

I found the this error:
```
usage: ipykernel_launcher.py [-h] [-config CONFIG] [-save_config SAVE_CONFIG]
                             -data DATA
                             [-skip_empty_level {silent,warning,error}]
                             [-transforms {filtertoolong,prefix,switchout,tokendrop,tokenmask,bart,sentencepiece,bpe,onmt_tokenize} [{filtertoolong,prefix,switchout,tokendrop,tokenmask,bart,sentencepiece,bpe,onmt_tokenize} ...]]
                             -save_data SAVE_DATA [-overwrite]
                             [-n_sample N_SAMPLE] [-dump_samples]
                             [-num_threads NUM_THREADS]
                             [-vocab_sample_queue_size VOCAB_SAMPLE_QUEUE_SIZE]
                             -src_vocab SRC_VOCAB [-tgt_vocab TGT_VOCAB]
                             [-share_vocab] [--src_seq_length SRC_SEQ_LENGTH]
                             [--tgt_seq_length TGT_SEQ_LENGTH]
                             [-switchout_temperature SWITCHOUT_TEMPERATURE]
                             [-tokendrop_temperature TOKENDROP_TEMPERATURE]
                             [-tokenmask_temperature TOKENMASK_TEMPERATURE]
                             [--permute_sent_ratio PERMUTE_SENT_RATIO]
                             [--rotate_ratio ROTATE_RATIO]
                             [--insert_ratio INSERT_RATIO]
                             [--random_ratio RANDOM_RATIO]
                             [--mask_ratio MASK_RATIO]
                             [--mask_length {subword,word,span-poisson}]
                             [--poisson_lambda POISSON_LAMBDA]
                             [--replace_length {-1,0,1}]
                             [-src_subword_model SRC_SUBWORD_MODEL]
                             [-tgt_subword_model TGT_SUBWORD_MODEL]
                             [-src_subword_nbest SRC_SUBWORD_NBEST]
                             [-tgt_subword_nbest TGT_SUBWORD_NBEST]
                             [-src_subword_alpha SRC_SUBWORD_ALPHA]
                             [-tgt_subword_alpha TGT_SUBWORD_ALPHA]
                             [-src_subword_vocab SRC_SUBWORD_VOCAB]
                             [-tgt_subword_vocab TGT_SUBWORD_VOCAB]
                             [-src_vocab_threshold SRC_VOCAB_THRESHOLD]
                             [-tgt_vocab_threshold TGT_VOCAB_THRESHOLD]
                             [-src_subword_type {none,sentencepiece,bpe}]
                             [-tgt_subword_type {none,sentencepiece,bpe}]
                             [-src_onmttok_kwargs SRC_ONMTTOK_KWARGS]
                             [-tgt_onmttok_kwargs TGT_ONMTTOK_KWARGS]
                             [--seed SEED]
ipykernel_launcher.py: error: the following arguments are required: -src_vocab/--src_vocab

 SystemExit: 2 

/opt/conda/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3426: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.
  warn("To exit: use 'exit', 'quit', or Ctrl-D.", stacklevel=1)
```

**Comment by francoishernandez** [comment]
Yes, this example was not updated following #1878. You should actually add the `src_vocab` and `tgt_vocab` opts in the build_vocab command.

**Comment by ishaansharma** [comment]
You mean something like this:
```
base_args = ([ "-config", "toy-ende/config.yaml","-n_sample", "10000","-src_vocab","path_of_src_vocab","-tgt_vocab","path_of_tgt_vocab"])
opts, unknown = parser.parse_known_args(base_args)
```

**Comment by francoishernandez** [comment]
There may be an issue in your paths somewhere.
Anyways, the fix in #1939 is working (just tested).
Note: [the docs were correct](https://opennmt.net/OpenNMT-py/examples/Library.html#prepare-data-and-vocab) but the notebook was not updated.
